Sources/MySQL/Bind.swift

```diff
@@ -18,7 +22,7 @@ public final class Bind {
     public typealias CBind = MYSQL_BIND
 
     /// MySQL represents Characters using signed integers.
```

Comment:
It would be great to get this merged. I need to make some changes to my fork but they'll affect this merge if I push them right now...